core-bundle/src/Resources/contao/classes/BackendTemplate.php

```diff
@@ -134,6 +134,17 @@ protected function compile()
 			$this->mootools .= $strMootools;
 		}
 
+		// Add environment name to template @see #403
+		if (System::getContainer()->hasParameter('contao'))
```

Comment:
Why not just `contao.environment`?

Reply:
Weil es eine Fehlermeldung gibt, wenn kein Wert gesetzt ist, was ja der Standardfall ist.

Reply:
Ja der `hasParameter()` Aufruf ist ja in Ordnung, aber `contao` gibt's eh nicht als Parameter. Das wird immer failen.